tests/drivers/i2s/i2s_speed/src/test_i2s_speed.c

```diff
@@ -540,3 +495,61 @@ ZTEST(drivers_i2s_speed, test_i2s_dir_both_transfer_long)
 	}
 	zassert_equal(num_verified, NUM_BLOCKS, "Invalid RX blocks received");
 }
+
+void *test_i2s_speed_configure(void)
+{
+	/* Configure I2S TX transfer. */
+	int ret;
+
+	dev_i2s_tx = DEVICE_DT_GET_OR_NULL(I2S_DEV_NODE_TX);
+	zassert_not_null(dev_i2s_tx, "transfer device not found");
+	zassert(device_is_ready(dev_i2s_tx), "transfer device not ready");
+
+	ret = configure_stream(dev_i2s_tx, I2S_DIR_TX);
+	zassert_equal(ret, TC_PASS);
+
+	/* Configure I2S RX transfer. */
+	dev_i2s_rx = DEVICE_DT_GET_OR_NULL(I2S_DEV_NODE_RX);
+	zassert_not_null(dev_i2s_rx, "receive device not found");
+	zassert(device_is_ready(dev_i2s_rx), "receive device not ready");
+
+	ret = configure_stream(dev_i2s_rx, I2S_DIR_RX);
+	zassert_equal(ret, TC_PASS);
+
+	return 0;
+}
+
+void *test_i2s_speed_rxtx_configure(void)
+{
+	/* Configure I2S TX transfer. */
+	int ret;
+
+	dev_i2s_tx = DEVICE_DT_GET_OR_NULL(I2S_DEV_NODE_TX);
+	zassert_not_null(dev_i2s_tx, "transfer device not found");
+	zassert(device_is_ready(dev_i2s_tx), "transfer device not ready");
+
+	ret = configure_stream(dev_i2s_tx, I2S_DIR_TX);
+	zassert_equal(ret, TC_PASS);
```

Comment:
This is needless. `dev_i2s_tx` is not used in `_txrx` tests.

Reply:
Thank you, I will update this.